nucypher/blockchain/eth/actors.py

```diff
@@ -484,20 +495,24 @@ def initialize_stake(self,
 
         if expiration:
             lock_periods = calculate_period_duration(future_time=expiration)
+
         if entire_balance is True:
             amount = self.token_balance
 
+        amount = NU(int(amount), 'NUWei')
```

Comment:
Why the NU -> int -> NU conversion here?

Reply:
This is handling two input types: An `int` or  a `NU` instance, depending on weather or not this input was accepted interactively or via the `--value` flag.

Reply:
👍